demisto_sdk/commands/init/initiator.py

```diff
@@ -197,14 +233,15 @@ def __init__(
         category: str = "",
         script: bool = False,
         pack: bool = False,
+        xsiam: bool = False,
         author_image: str = "",
         demisto_mock: bool = False,
         common_server: bool = False,
         **kwargs,
     ):
         self.output = output if output else ""
         self.id = id
-
+        self.xsiam = xsiam
```

Comment:
use `marketplace` instead, of type `MarketplaceVersions`, rather than string. See `uploader.py` for reference.